test/parallel/test-cluster-worker-deprecated.js

```diff
@@ -7,12 +7,12 @@ const cluster = require('cluster');
 const worker = new cluster.Worker();
 
 assert.strictEqual(worker.exitedAfterDisconnect, undefined);
-assert.strictEqual(worker.suicide, undefined);
+assert.strictEqual(worker.exitedAfterDisconnect, undefined);
```

Comment:
@gibfahn Yes, I agree, it seems like this test file can be removed rather than modified.